jenkinsfile-runner/README.md

```diff
@@ -0,0 +1,52 @@
+# Jenkinsfile Runner
+
+Dockerfile for an image with [Jenkinsfile Runner](https://github.com/jenkinsci/jenkinsfile-runner) based on `ppiper/jenkins-master`.
+
+This allows to execute a `Jenkinsfile` from the command-line, thus without the need for a long-running, stateful Jenkins master instance.
+
+## Download
+
+This image is published to Docker Hub and can be pulled via the command
+
+```
+docker pull ppiper/jenkinsfile-runner
+```
+
+## Build
+
+To build this image locally, open a terminal in the directory of the Dockerfile an run
+
+```
+docker build -t ppiper/jenkinsfile-runner .
+```
+
+**Warning:** Building this image will take quite long, as the Jenkinsfile Runner is built from source and has many dependencies.
+Expect high network traffic.
```

Comment:
See comment above

Reply:
Regardless of cache, the first build will always take long and download lots of dependencies. Also, Docker does cache the layer, so I'm not really sure if I understand the benefit of what they do in the upstream Dockerfile. I'd wanted to keep it simple. What is the benefit of what they do?